vector-im/element-android
# [Device Manager] Filter Other Sessions (PSG-684)

## Type of change


## Content


Create a dedicated screen to list other sessions. Add the ability to filter verified, unverified and inactive sessions.

## Motivation and context

Fixes #7045.

## Screenshots / GIFs



https://user-images.githubusercontent.com/1254401/189923219-7909363a-68b8-4c6c-b174-aa2b450f7fee.mp4




## Tests



- Enable `VectorFeatures.isNewDeviceManagementEnabled`
- Navigate to Settings > Security & Privacy > Show All Sessions (V2, WIP)
- Click on View All button under the Other sessions section
- See the list of all other sections
- Click on the filter icon on navigation bar. Test different filters.
- Filter icon should have a green badge if there is an active filter.
- "Clear filter" option should appear if there is not any session according to the active filter.

## Tested devices

- OS version(s):

## Checklist
